developerportal/templatetags/app_tags.py

```diff
@@ -124,22 +126,27 @@ def pagination_additional_filter_params(request):
     """
 
     QUERY_PARAMS_TO_PASS_ON = (
-        # PAGINATION_QUERYSTRING_KEY,  # we DON'T want the page param
+        # PAGINATION_QUERYSTRING_KEY,  # we DON'T want this one: it's added separately
         TOPIC_QUERYSTRING_KEY,
         ROLE_QUERYSTRING_KEY,
         COUNTRY_QUERYSTRING_KEY,
         DATE_PARAMS_QUERYSTRING_KEY,
+        SEARCH_QUERYSTRING_KEY,
     )
 
     output_params_strings = []
 
     input_keys = request.GET.keys()
+
     for input_key in input_keys:
         if input_key not in QUERY_PARAMS_TO_PASS_ON:
             continue
         input_params = request.GET.getlist(input_key)
         for input_param in input_params:
-            output_params_strings.append(f"{input_key}={input_param}")
+            # Note that using `urllib.parse.quote` avoids spaces appearing in
+            # the output, which breaks Safari (because the space triggers a web
+            # search, not a HTTP request to the site)
+            output_params_strings.append(f"{input_key}={quote(input_param)}")
```

Comment:
I went with `quote` rather than `quote_plus`, but can't really pick a firm reason to stand on this hill. What do you think?

Reply:
Note: maybe it doesn't make a huge difference, as it'll be mitigated in the FE code in the next PR, because Firefox plus-quotes anyway

Reply:
I'm erring towards quote_plus, but don't have a good reason for it yet (I like reading those kinds of URLs better. Google does it.. I guess those are semi-okay reasons).

Reply:
Cheers. Going to go with `quote_plus` because it matches the behaviour of `x-www-form-urlencoded`, which is what is going on here (it's a form submission, after all)